rules/cross-platform/initial-access-zoom-meeting-with-no-passcode.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/09/14"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies Zoom meetings that are created without a passcode. Meetings without a passcode are susceptible to
+Zoombombing. Zoombombing is carried out by taking advantage of Zoom sessions that are not protected with a passcode.
+Zoombombing refers to the unwanted, disruptive intrusion, generally by Internet trolls and hackers, into a video
+conference call. In a typical Zoombombing incident, a teleconferencing session is hijacked by the insertion of material
+that is lewd, obscene, racist, or antisemitic in nature, typically resulting of the shutdown of the session.
```

Comment:
```suggestion
Identifies Zoom meetings that are created without a passcode. Meetings without a passcode are susceptible to
Zoombombing. Zoombombing refers to the unwanted, disruptive intrusion, generally by Internet trolls and hackers, into a video
conference call. In a typical Zoombombing incident, a teleconferencing session is hijacked by the insertion of material
that is lewd, obscene, racist, or antisemitic in nature, typically resulting of the shutdown of the session.
```

Reply:
I think the below was repetitive, so suggested taking out the second sentence. You'll most likely need to re-lint if accepting the suggestion. 
> Meetings without a passcode are susceptible to Zoombombing. Zoombombing is carried out by taking advantage of Zoom sessions that are not protected with a passcode.